src/App.js

```diff
@@ -1,26 +1,104 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
+import Search from './components/Search'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+
+
+  state = {
+    foods: foods,
+    filteredFoods: foods,
+    todayFoods: [],
+    wantsToAdd: false
+  }
+
+  handleAddFood = (event) => {
+    event.preventDefault()
+    let foodName = event.currentTarget.food.value
+    let foodCal = event.currentTarget.calories.value
+    let foodImg = event.currentTarget.image.value
+    let newFoods = [{name: foodName, calories: foodCal, image: foodImg}, ...this.state.foods]
+    this.setState({
+      foods: newFoods,
+      wantsToAdd: !this.state.wantsToAdd
+    })
+  }
+
+  handleShowAdd = () => {
+    this.setState({
+      wantsToAdd: !this.state.wantsToAdd
+    })
+  }
+
+  handleSearch = (event) => {
+    let searchFood = event.currentTarget.value;
+    let cloneFoods = this.state.foods.filter((food) => {
+      return food.name.toLowerCase().includes(searchFood.toLowerCase())
+    })
+    this.setState({
+      filteredFoods: cloneFoods
+    })
+  }
+
+
+  handleAddToToday = (id, quantity) => {
+    let foodToAdd = this.state.foods[id]
+    foodToAdd.quantity = quantity;
+
+
+  
+    this.setState({
+      todayFoods: [...this.state.todayFoods, foodToAdd],
+
+    })
+  }
+  
+
+
+
+
+
+
+  render(){
+    return (
+      <div className="App">
+      <div className="columns">
+      <div className="column">
+        {this.state.wantsToAdd && <AddFood onAdd={this.handleAddFood}/>}
+        {!this.state.wantsToAdd && <button onClick={this.handleShowAdd} 
```

Comment:
if you want to show different things based on a boolean then a ternary is a better approach.

`this.state.wantsToAdd ? <AddFood onAdd={this.handleAddFood}/> : <button onClick={this.handleShowAdd} className="button">Add Food</button>`